awx_collection/test/awx/test_inventory_source.py

```diff
@@ -42,6 +42,7 @@ def test_inventory_source_create(run_module, admin_user, base_inventory, project
     assert result == {
         'id': inv_src.id,
         'name': 'foo',
+        'created': True,
```

Comment:
just change the whole thing to `assert result['id'] == inv_src.id` because it is being said here that the others are not supported.

The name return value might be alright, but it's not given in all cases. No one has any idea if should be there or not.